samples/subsys/lw_sched/README.rst

```diff
@@ -0,0 +1,59 @@
+.. zephyr:code-sample:: lw_sched
+   :name: Lightweight (LW) Scheduler
```

Comment:
nit: unless it's like a proper name, or a well-known acronym i'd like to keep the sample names non capitalized. Looks like you're mostly spelling it in lower case yourself anyway!

```suggestion
   :name: Lightweight (LW) scheduler
```

Reply:
Fixing.